pkgs/by-name/lo/lora/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lora";
+  version = "v3.005";
+
+  src = fetchFromGitHub {
+    owner = "cyrealtype";
+    repo = "lora";
+    rev = version;
```

```suggestion
    rev = finalAttrs.version;
```
- #315337
- https://nix.dev/guides/best-practices#recursive-attribute-set-rec